src/diffusers/pipelines/latte/pipeline_latte.py

```diff
@@ -0,0 +1,762 @@
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import html
+import inspect
+import re
+import urllib.parse as ul
+from typing import Callable, List, Optional, Tuple, Union
+
+import torch
+from transformers import T5EncoderModel, T5Tokenizer
+from ...video_processor import VideoProcessor
+from ...models import AutoencoderKL, LatteTransformer3DModel
+from ...schedulers import DDIMScheduler
+from ...utils import (
+    BACKENDS_MAPPING,
+    is_bs4_available,
+    is_ftfy_available,
+    logging,
+    replace_example_docstring,
+    BaseOutput,
+)
+from ...utils.torch_utils import is_compiled_module, randn_tensor
+from ...pipelines.pipeline_utils import DiffusionPipeline
+from dataclasses import dataclass
+
+logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
+
+if is_bs4_available():
+    from bs4 import BeautifulSoup
+
+if is_ftfy_available():
+    import ftfy
+
+
+EXAMPLE_DOC_STRING = """
+    Examples:
+        ```py
+        >>> import torch
+        >>> import imageio
+        >>> from diffusers import LattePipeline
+
+        >>> # You can replace the checkpoint id with "maxin-cn/Latte-1" too.
+        >>> pipe = LattePipeline.from_pretrained("maxin-cn/Latte-1", torch_dtype=torch.float16)
+        >>> # Enable memory optimizations.
+        >>> pipe.enable_model_cpu_offload()
+
+        >>> prompt = "A small cactus with a happy face in the Sahara desert."
+        >>> videos = pipe(prompt).video
+        >>> imageio.mimwrite('./test_pipeline.mp4', videos[0], fps=8, quality=5) # highest quality is 10, lowest is 0
+        ```
+"""
+
+@dataclass
+class LattePipelineOutput(BaseOutput):
+    video: torch.Tensor
```

Comment:
we normally use `frames` I think https://github.com/huggingface/diffusers/blob/57084dacc5275d7212513b24837b60a28e55603d/src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py#L136

Reply:
Okay. I changed it.